frontend/src/component/feature/CreateFeature/CreateFeature.test.tsx

```diff
@@ -28,50 +28,66 @@ const setupApi = ({
     });
 };
 
-test("should allow you to create feature flags when you're below the global limit", async () => {
-    setupApi({ flagLimit: 3, flagCount: 2 });
+describe('button states', () => {
+    test("should allow you to create feature flags when you're below the global limit", async () => {
+        setupApi({ flagLimit: 3, flagCount: 2 });
 
-    render(
-        <Routes>
-            <Route
-                path='/projects/:projectId/create-toggle'
-                element={<CreateFeature />}
-            />
-        </Routes>,
-        {
-            route: '/projects/default/create-toggle',
-            permissions: [{ permission: CREATE_FEATURE }],
-        },
-    );
+        render(
+            <Routes>
+                <Route
+                    path='/projects/:projectId/create-toggle'
+                    element={<CreateFeature />}
+                />
+            </Routes>,
+            {
+                route: '/projects/default/create-toggle',
+                permissions: [{ permission: CREATE_FEATURE }],
+            },
+        );
 
-    await waitFor(async () => {
         const button = await screen.findByRole('button', {
             name: /create feature flag/i,
         });
-        expect(button).not.toBeDisabled();
+        await waitFor(() => {
+            expect(button).not.toBeDisabled();
+        });
     });
 });
 
-test("should not allow you to create API tokens when you're at the global limit", async () => {
```

Comment:
This test was always true because the button starts off as disabled when we're loading, so I've removed it.